cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -830,79 +830,142 @@ values will be stored in the Join Server.`,
 			return io.Write(os.Stdout, config.OutputFormat, ids)
 		},
 	}
+	endDevicesListQRCodeFormatsCommand = &cobra.Command{
+		Use:     "list-qr-formats",
+		Aliases: []string{"lsqrfmts", "lsqrfmt"},
```

Comment:
For consistency with `endDeviceTemplatesListFormats`, this would be `[]string{"ls-qr-formats", "listqrformats", "lsqrformats"},`. 

You may also want to add `fmts`, `fmt` to the other one.

Also consider `endDevicesListFrequencyPlans` and `gatewaysListFrequencyPlans`